### ISSUE TITLE
The problem of hyperparameter setting, AdaAD_alpha

### ISSUE THREAD

**Issue by ThreeTree323** [boyellow/AdaAD#5]
In the paper, AdaAD_alpha has an adaptive function, but in the main.py code, the AdaAD_alpha parameter is fixed at "1". Is this due to incomplete code?

    parser.add_argument('--AdaAD_alpha', default=1.0, type=float, help='AdaAD_alpha')
            loss = args.AdaAD_alpha*kl_loss1 + (1-args.AdaAD_alpha)*kl_loss2
